### PR TITLE
add library pros and cons to README

### PR DESCRIPTION
**Pull request by SethTisue**
Sometimes people stumble upon this library and figure that if it it's in the scala org it must be the bee's knees. I think we should foreground the reasons to choose it or not.

### CODE REVIEW

**Comment by SethTisue** [comment]
@j-mie6 @johnynek LGTY? is anything major missing from the "weaknesses" section? accent on "major" — that section shouldn't be too long and needn't be comprehensive

**Comment by j-mie6** [comment]
I'd say that:

* The API is a little disorganised and missing a few niceties that I'd expect from a modern PC library (like precedence combinators)
* it doesn't seem to have any way of customising the error messages/formatting other than raw messages
* it doesn't have any natural integration with something like `cats`?

**Comment by SethTisue** [comment]
Those are good observations, but maybe a bit too detailed. I added a bullet that says "Minimal feature set" in an attempt  to cover it.

**Comment by j-mie6** [comment]
Perhaps "Rigid error messages" might be enough high-level?

**Comment by SethTisue** [comment]
(Incidentally, I wouldn't be opposed to linking to external resources with more detailed comparison info.)

**Comment by SethTisue** [comment]
Added: "Inflexible, unstructured error reporting"

File: README.md [hunk 1]
```diff
@@ -4,9 +4,23 @@
...
+This was originally part of the Scala standard library, but is now community-maintained, under the guidance of the Scala team at Lightbend. If you are interested in joining the maintainers team, please contact [@Philippus](https://github.com/philippus) or [@SethTisue](https://github.com/SethTisue).
 
-This library was originally part of the Scala standard library, but is now community-maintained, under the guidance of the Scala team at Lightbend. If you are interested in helping please contact [@Philippus](https://github.com/philippus) or [@SethTisue](https://github.com/SethTisue).
+## Choosing a parsing library
+
+This library's main strengths are:
+
+* Stability. It's been around and in wide use for more than a decade.
+* The codebase is modest in size and its internals are fairly simple.
+* It's plain vanilla Scala. No macros, code generation, or other magic is involved.
```
Comment (johnynek) [inline README.md:15]: I'd probably add that there seem to be versions of this library for all versions of scala (including js and native), while several of the alternatives can't make that claim.

**Comment by SethTisue** [comment]
merging... we can always change it further. thanks for the help

**Pull request merged**